nion/swift/ExportDialog.py

```diff
@@ -423,7 +438,12 @@ def __init__(self, document_controller: DocumentController.DocumentController,
         super().__init__()
         self.__document_controller = document_controller
         self.__display_item = display_item
-        self.__model = ExportSizeModel(display_item)
+        self.__units_model = UserInterface.StringPersistentModel(
```

Comment:
Can you create the persistent model using `ui.create_persistent_string_model` instead of instantiating it directly to be consistent with other similar usages throughout the code. Also can you qualify the storage_key further, something like `export_svg_dialog_units` instead of `export_units`?